### ISSUE TITLE
How to Free Geany's CTRL-A/E?

### ISSUE THREAD

**Issue by sami10007** [geany/geany#1061]
I would like to use the default terminal keybindings. 

Geanys
- CTRL-A select all
- CTRL-E comment current line

I want defaults 
- CTRL-A go to the beginning of the line
- CTRL-E go to the end of the line
- CTRL-/ comment the line

One liner would be great to adjust these settings for future installations. 

How can you Free Geany's CTRL-A/E?

**Comment by b4n** [comment]
*Override Geany keybindings* setting is likely what you want, see https://www.geany.org/manual/#terminal-vte-preferences

**Comment by b4n** [comment]
And you of course can change the key bindings that currently use <kbd>Ctrl+A</kbd> (*Selection/Select all*) and <kbd>Ctrl+E</kbd> (*Format/Comment/Uncomment line*) in the preferences.

**Comment by sami10007** [comment]
Is there any terminal command to change those things? It would really save my time to configure the editor in many environments.

**Comment by elextr** [comment]
@sami10007 no, you have to change them in Geany, but they are saved so you only need to do it once.

**Comment by b4n** [comment]
And you can copy the configuration files over to different machines.
